UM/Resources.py

```diff
@@ -264,55 +268,188 @@ def __find(cls, resource_type: int, *args) -> List[str]:
         return files
 
     @classmethod
-    def __initializeStoragePaths(cls):
-        # use nested structure: cura/<version>/...
-        if cls.ApplicationVersion == "master" or cls.ApplicationVersion == "unknown":
-            storage_dir_name = os.path.join(cls.ApplicationIdentifier, cls.ApplicationVersion)
-        else:
-            from UM.Version import Version
-            version = Version(cls.ApplicationVersion)
-            storage_dir_name = os.path.join(cls.ApplicationIdentifier, "%s.%s" % (version.getMajor(), version.getMinor()))
-
+    def getConfigStorageRootPath(cls):
```

Comment:
This exposes an abbreviation as public API. (Config vs. Configuration) We generally try not use abbreviations, and should definitely not expose them as public API.